catalog/security/catalog-security-filter/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -14,8 +14,22 @@
 -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
-    <bean id="filterPlugin" class="ddf.catalog.security.filter.plugin.FilterPlugin"/>
+    <bean id="filterStrategySortedList" class="ddf.catalog.util.impl.SortedServiceList" />
+
+    <reference-list id="filterStrategies" interface="ddf.catalog.security.FilterStrategy"
+                    availability="optional">
+        <reference-listener bind-method="bindPlugin"
+                            unbind-method="unbindPlugin" ref="filterStrategySortedList"/>
+    </reference-list>
+
+    <bean id="filterPlugin" class="ddf.catalog.security.filter.plugin.FilterPlugin">
+        <argument ref="filterStrategySortedList" />
+    </bean>
+
+    <bean id="filteringStrategy" class="ddf.catalog.security.filter.strategy.FilteringStrategy" />
 
     <service ref="filterPlugin" auto-export="interfaces" ranking="0"/>
 
+    <service ref="filteringStrategy" auto-export="interfaces" ranking="100" />
```

Comment:
I see that this blueprint already has `auto-export` but in general I am not a fan and prefer being explicit about the interfaces you want to export.